.gitignore

```diff
@@ -19,3 +19,4 @@ packages/*/package-lock.json
 !**/fixtures/**/plugins_cache*/.netlify
 !**/fixtures/**/pin_*/.netlify
 !**/fixtures/**/functions_*/.netlify
+packages/build/tests/core/fixtures/zip_too_big/veryLargeFile.js
```

Comment:
Should the test delete this file in a `try`/`finally` block instead of adding it to `.gitignore`?

Reply:
agreed! 03c3a76